.github/workflows/rubyonrails.yml

```diff
@@ -70,12 +70,10 @@ jobs:
       DISPLAY: "=:99"
 
     steps:
-      - uses: nanasess/setup-chromedriver@v2
-      - name: Start chromedriver
-        run: |
-          set -x
-          chromedriver --url-base=/wd/hub &
-          # sudo Xvfb -ac :99 -screen 0 1280x1024x24 > /dev/null 2>&1 & # optional
+      - name: Setup Chrome
+        uses: browser-actions/setup-chrome@v1.7.2
+        with:
+          install-dependencies: true
```

Comment:
Interesting, what's the difference between these two ways of doing system tests?

Also, the original way looks a little bit odd. I found the original PR in which Rob touched on this, but I don't fully understand his comment. Maybe this means something to you? https://github.com/AllYourBot/hostedgpt/pull/66